src/api/env.development

```diff
@@ -13,6 +13,24 @@ API_HOST=api.telescope.localhost
 # The API Version, used as a prefix on all routes: /v1
 API_VERSION=v1
 
+# Redis info
+REDIS_URL=http://redis:6379
+REDIS_PORT=6379
```

Comment:
Let's simplify the code and just pass the URL in, removing port and password, since we don't mess with either in Docker.